flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/source/IcebergTableSource.java

```diff
@@ -84,7 +84,7 @@ public IcebergTableSource(
       TableSchema schema,
       Map<String, String> properties,
       ReadableConfig readableConfig) {
-    this(loader, schema, properties, null, false, -1, ImmutableList.of(), readableConfig);
+    this(loader, schema, properties, null, false, null, ImmutableList.of(), readableConfig);
```

Comment:
similar to the other comment. this doesn't seem right. it will also be inconsistent with the doc which said the default is `-1`.

Reply:
The value here overrides flink config. It should set a default value `null`, and then the read conf take the default value -1 when no one(options\config) is used.

Reply:
got it. thx